daemon/daemon_unix.go

```diff
@@ -1151,3 +1154,20 @@ func setupOOMScoreAdj(score int) error {
 	f.Close()
 	return err
 }
+
+func (daemon *Daemon) setupSeccompProfile() error {
+	b, err := ioutil.ReadFile(seccompProfilePath)
+	if err != nil {
+		// TODO(runcom): generate a default seccomp profile if system wide isn't present?
```

Comment:
I don't think we should generate one either, as that would mean embedding the json in the binary which seems gross.

Does the CI use `make install`? In which case can fix that.